### PR TITLE
fix backstage manifests

### PR DESCRIPTION
**Pull request by nabuskey**
Updated the backstage manifests to:

1. Use in-memory database
2. Setup cluster role and binding to work with the default backstage image
3. Make the kind cluster available as a target cluster in Backstage

fixes: #12

### CODE REVIEW

File: pkg/apps/srv/backstage/install.yaml [hunk 1]
```diff
@@ -90,15 +78,93 @@ spec:
...
+    auth:
+      environment: local
+      providers: {}
+    scaffolder: {}
+    catalog:
+      import:
+        entityFilename: catalog-info.yaml
+        pullRequestBranchName: backstage-integration
+      rules:
+        - allow: [Component, System, API, Resource, Location, Template]
```
Comment (blakeromano) [inline pkg/apps/srv/backstage/install.yaml:154]: is there a reason why are we not allowing domains into backstage?

**Review by jessesanford — APPROVED** [review]
LGTM

File: pkg/apps/srv/backstage/install.yaml (hunk 1 above)
Comment (nabuskey) [reply pkg/apps/srv/backstage/install.yaml:154]: Good point, really no reason to prohibit it for local developments.

Comment (blakeromano) [reply pkg/apps/srv/backstage/install.yaml:154]: Although in general I could see us wanting to allow customization on things like `entityFilename` where some organizations use custom file names so I wonder if some of this being done through a GoLang template may make sense in the future.

Comment (nabuskey) [reply pkg/apps/srv/backstage/install.yaml]: Yea we can allow for more customization in the future. These manifests allow you to get started quickly locally.

**Review by nimakaviani — COMMENTED** [review]
Shall we add ingress too, now that we are fixing it all?

**Pull request merged**